homeassistant/components/fujitsu_hvac/config_flow.py

```diff
@@ -0,0 +1,95 @@
+"""Config flow for Fujitsu HVAC (based on Ayla IOT) integration."""
+
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+from ayla_iot_unofficial import AylaAuthError, new_ayla_api
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry, ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import API_TIMEOUT, CONF_EUROPE, DOMAIN, FGLAIR_APP_ID, FGLAIR_APP_SECRET
+
+_LOGGER = logging.getLogger(__name__)
+
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_EUROPE): bool,
+    }
+)
+
+
+class FujitsuHVACConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Fujitsu HVAC (based on Ayla IOT)."""
+
+    reauth_entry: ConfigEntry | None = None
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle the initial step."""
+        errors: dict[str, str] = {}
+        if user_input:
+            if not self.reauth_entry:
+                await self.async_set_unique_id(user_input[CONF_USERNAME].lower())
+                self._abort_if_unique_id_configured()
+            elif user_input[CONF_USERNAME] != self.reauth_entry.data[CONF_USERNAME]:
```

Comment:
One thing I am wondering, can someone change username? I think it would be a lot better if the reauth form is only the password field, since changing username or the Europe setting isn't reauth concern